lambda/iam-notify-slack.py

```diff
@@ -59,74 +57,83 @@ def lambda_handler(event, context):
+                5]
             policy_arn = event['detail']['requestParameters']['policyArn']
-        elif event_name == "AttachUserPolicy" or event_name == "DetachUserPolicy":
+        elif event_name == "AttachUserPolicy" or event_name == "DetachUserPolicy":  # noqa: E501
             post_to_slack = True
             object_field_name = "User"
-            object_field_value = event['detail']['requestParameters']['userName']
-            policy_name = event['detail']['requestParameters']['policyArn'].split(':')[5]
+            object_field_value = event['detail']['requestParameters']['userName']  # noqa: E501
+            policy_name = event['detail']['requestParameters']['policyArn'].split(':')[  # noqa: E501
+                5]
             policy_arn = event['detail']['requestParameters']['policyArn']
-        elif event_name == "AttachRolePolicy" or event_name == "DetachRolePolicy":
+        elif event_name == "AttachRolePolicy" or event_name == "DetachRolePolicy":  # noqa: E501
             post_to_slack = True
             object_field_name = "Role"
-            object_field_value = event['detail']['requestParameters']['roleName']
-            policy_name = event['detail']['requestParameters']['policyArn'].split(':')[5]
+            object_field_value = event['detail']['requestParameters']['roleName']  # noqa: E501
+            policy_name = event['detail']['requestParameters']['policyArn'].split(':')[  # noqa: E501
+                5]
             policy_arn = event['detail']['requestParameters']['policyArn']
         else:
-            print "No support for event " + event_name
+            print("No support for event " + event_name)
             print("Received event: " + json.dumps(event, indent=2))
 
         if post_to_slack:
-            print "Posting to slack for " + event_name
-            print "object_field_name " + object_field_name
-            print "object_field_value " + object_field_value
-            print "policy_name " + policy_name
-            print "policy_arn " + policy_arn
+            print("Posting to slack for " + event_name)
+            print("object_field_name " + object_field_name)
+            print("object_field_value " + object_field_value)
+            print("policy_name " + policy_name)
+            print("policy_arn " + policy_arn)
 
             # Get the user or role who made this change
             if 'userName' in event['detail']['userIdentity']:
                 operation_user = event['detail']['userIdentity']['userName']
             else:
                 # no user so must be a role
-                operation_user = event['detail']['userIdentity']['principalId'].split(':')[1]
-                operation_role = event['detail']['userIdentity']['sessionContext']['sessionIssuer']['userName']
+                operation_user = event['detail']['userIdentity']['principalId'].split(':')[  # noqa: E501
+                    1]
```

Comment:
This change seems to make things less readable to me... to only split the 1] onto a separate line